flashlight/app/asr/criterion/CMakeLists.txt

```diff
@@ -18,6 +18,8 @@ if (FL_USE_CUDA)
   # cuRAND is required and linked below
   set(WARPCTC_INCLUDE_DIR ${WARPCTC_DIR}/include)
   target_include_directories(flashlight-app-asr PRIVATE ${WARPCTC_INCLUDE_DIR})
+  # Required when using cmmake 3.12+:
+  target_compile_options(flashlight-app-asr PRIVATE $<$<COMPILE_LANGUAGE:CUDA>:--std=c++11>)
```

Comment:
we switch now to c++14, could you fix with respect to this change?

Reply:
done